.eslintrc

```diff
@@ -42,8 +42,9 @@
     "linebreak-style": ["error", "unix"],
     "react/no-array-index-key": "off",
     "react/react-in-jsx-scope": "off",
-    "@typescript-eslint/no-var-requires": "off",
-    "@typescript-eslint/no-unused-vars": "off",
+    "@typescript-eslint/no-var-requires": "error",
+    "@typescript-eslint/no-unused-vars": "error",
```

Comment:
this overrides rule below unused-imports/no-unused-vars

Reply:
this TS rule not JS